renku/models/_jsonld.py

```diff
@@ -372,17 +370,17 @@ def from_jsonld(
         return self
 
     @classmethod
-    def from_yaml(cls, path, client=None):
+    def from_yaml(cls, path, **kwargs):
```

Comment:
I generally think kwargs should only be used as a last resort. Can we be explicit with our params or at least document what can come in kwargs?

Reply:
Sure, it was just starting to look wordy but you're probably right